diff --git a/modules/luci-mod-status/htdocs/luci-static/resources/view/status/index.js b/modules/luci-mod-status/htdocs/luci-static/resources/view/status/index.js
old mode 100644
new mode 100755
index 5f77845..8789047
--- a/modules/luci-mod-status/htdocs/luci-static/resources/view/status/index.js
+++ b/modules/luci-mod-status/htdocs/luci-static/resources/view/status/index.js
@@ -286,7 +286,9 @@ return view.extend({
 			{ name: 'view.status.include.30_network' },
 			{ name: 'view.status.include.40_dhcp', title: _('DHCP Leases'), deferFirstLoad: true },
 			{ name: 'view.status.include.50_dsl' },
-			{ name: 'view.status.include.60_wifi' }
+			{ name: 'view.status.include.60_wifi' },
+			{ name: 'view.status.include.95_homestatus_disks' }, /* HOMELEDE-CUSTOM */
+			{ name: 'view.status.include.95_homestatus_apps' } /* HOMELEDE-CUSTOM */
 		];
 
 		return Promise.all(includeModules.map(L.bind(function(spec) {
